script.js

```diff
@@ -0,0 +1,10 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
```

Comment:
* As mentioned by the previous reviewer your drop-down menu is still different from the given design Kindly work on it 🙏 
## Yours 
![image](https://user-images.githubusercontent.com/52497246/202372388-3e4abe14-0b0b-478f-8cce-a3c3dc9d9a8c.png)
## Figma design 
![image](https://user-images.githubusercontent.com/52497246/202372499-e0d92c51-72cb-4fb7-bd41-b3ee751d3926.png)